app-text/htmltidy/patches/tidy-5.6.0.patchset

```diff
@@ -0,0 +1,60 @@
+From c94f856349bcc0179c9b510f74769c26c4bef43c Mon Sep 17 00:00:00 2001
+From: Adrien Destugues <pulkomandy@pulkomandy.tk>
+Date: Thu, 21 Dec 2017 08:31:26 +0100
+Subject: Port to Haiku.
+
+
+diff --git a/include/tidyplatform.h b/include/tidyplatform.h
+index 28de468..ca169dc 100644
+--- a/include/tidyplatform.h
++++ b/include/tidyplatform.h
+@@ -315,6 +315,15 @@ extern "C" {
+ #  endif
+ #endif
+ 
++/* === Convenience defines for Haiku platforms === */
++
++#if defined(__HAIKU__)
++#  define HAIKU
++#  ifndef PLATFORM_NAME
++#    define PLATFORM_NAME "Haiku"
++#  endif
++#endif
++
+ /* === Convenience defines for Cygwin platforms === */
+ 
+ #if defined(__CYGWIN__)
+@@ -428,7 +437,7 @@ extern "C" {
+ #if PRESERVE_FILE_TIMES
+ 
+ #  ifndef HAS_FUTIME
+-#    if defined(CYGWIN_OS) || defined(BE_OS) || defined(OS2_OS) || defined(HPUX_OS) || defined(SOLARIS_OS) || defined(LINUX_OS) || defined(BSD_BASED_OS) || defined(MAC_OS) || defined(__MSL__) || defined(IRIX_OS) || defined(AIX_OS) || defined(__BORLANDC__) || defined(__GLIBC__)
++#    if defined(CYGWIN_OS) || defined(BE_OS) || defined(OS2_OS) || defined(HPUX_OS) || defined(SOLARIS_OS) || defined(LINUX_OS) || defined(BSD_BASED_OS) || defined(MAC_OS) || defined(__MSL__) || defined(IRIX_OS) || defined(AIX_OS) || defined(__BORLANDC__) || defined(__GLIBC__) || defined(__HAIKU__)
```

Comment:
I'd put `__HAIKU__` right after the name of the last OS in that list, `AIX_OS`, and before the remaining names, which are not related to particular OSes: `__BORLANDC__` and `__GLIBC__`.
```diff
-+#    if defined(CYGWIN_OS) || defined(BE_OS) || defined(OS2_OS) || defined(HPUX_OS) || defined(SOLARIS_OS) || defined(LINUX_OS) || defined(BSD_BASED_OS) || defined(MAC_OS) || defined(__MSL__) || defined(IRIX_OS) || defined(AIX_OS) || defined(__BORLANDC__) || defined(__GLIBC__) || defined(__HAIKU__)
++#    if defined(CYGWIN_OS) || defined(BE_OS) || defined(OS2_OS) || defined(HPUX_OS) || defined(SOLARIS_OS) || defined(LINUX_OS) || defined(BSD_BASED_OS) || defined(MAC_OS) || defined(__MSL__) || defined(IRIX_OS) || defined(AIX_OS) || defined(__HAIKU__) || defined(__BORLANDC__) || defined(__GLIBC__)
```